wiki/osu!_wiki/Contribution_guide/Best_practices/en.md

```diff
@@ -0,0 +1,83 @@
+3. Click `Run workflow` and fill in the options:
+
+![](img/github-actions-workflow-dialog.png "GitHub Actions Workflow - Run Workflow")
+
+- **Use workflow from**: name of the branch you want to sync. By default, it is set to `master`.
+- **Overwrite any changes in the target repository**:
+  - `true`: replace the contents of your branch with a clean copy of the `master` branch from `ppy/osu-wiki`.
+  - `false` (default): merge your changes with the ones on `ppy/osu-wiki`.
+- **Create a backup of your target branch**:
+  - `true`:  make a branch called `backup-{name of your branch}` before changing it.
+  - `false` (default): do not make any backups.
+
+4. Click the `Run Workflow` button and wait for the workflow to complete. If you're curious about how the tool works, click on the `Sync from osu! upstream` workflow task.
+
+![](img/github-actions-workflow-overview.png "GitHub Actions Workflow - Workflow Overview")
+
+## Making edits
+
+Within your fork of the osu! wiki, you are free to make any changes and save them. **Commits** are individual "save points" of the repository. **Branches** are workspaces, which let you switch between multiple versions of the repository. To make your workflow easier and keep the history of the wiki clean and free from noise, follow these guidelines:
+
+- [Sync the `master` branch](#syncing-the-fork).
+- Always start the work by creating a new branch off `master`, and only keep your changes in there. Give it a meaningful name, such as `update-staff-log`.<!-- https://www.atlassian.com/git/tutorials/comparing-workflows/forking-workflow is the explanation, but it doesn't really fit in here -->
+- Commit your work when you've made reasonably sized changes. It's better to commit an article as a whole rather than 10 small edits.
+- **Use short and meaningful commit messages**, as they let others know what's in the box. Something like `Rewrite the section about jump patterns` says a lot more than `Update en.md`.
+
+## Opening a pull request
+
+A pull request shows other people how your edits will affect the files. Add some information to your pull request to explain your intentions:
+
+- `Title`: a very short descriptive title for your changes in English, together with the article's name. In case of a translation, start with the two-letter language name of your translations in brackets. Examples:
+  - ``[FR] Add `BBCode` ``
+  - ``Update `Beatmapping` and `Beatmap/Difficulty` ``
+- `Description`: anything you want to signal to the maintainers and other potential reviewers. Examples:
+  - A short summary of the changes, especially if they affect several articles
+  - The pull request's completeness, or ideas related to it
+  - [Automatic resolution of relevant issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
+- Make sure to tick the `Allow edits from maintainers` checkbox—it will allow the wiki maintainers to help you improve the pull request when necessary
```

Comment:
```suggestion
- Make sure to tick the `Allow edits from maintainers` checkbox, as it will allow the wiki maintainers to help you improve the pull request when necessary
```